src/renderer.c

```diff
@@ -49,6 +49,7 @@ typedef struct RenFont {
   unsigned short max_height, baseline, height;
   ERenFontAntialiasing antialiasing;
   ERenFontHinting hinting;
+  bool smoothing;
```

Comment:
I really think this should be part of `style`. This way you'll also avoid having to change the signature of the functions.

Reply:
I mean it doesn't change too much, We can make it a part of style, but I didn't feel like smoothing was a styling option, more like a rendering/loading one.